index.html

```diff
@@ -0,0 +1,131 @@
+        alt="Original Trombones logo"
+      />
+      <nav id="nav-bar">
+        <a class="nav-link" href="#features">Features</a>
+        <a class="nav-link" href="#how-it-works">How It Works</a>
+        <a class="nav-link" href="#pricing">Pricing</a>
+      </nav>
+    </header>
+
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <input type="email" name="email" id="email" placeholder="Enter your email address" />
+      <input type="submit" value="Get Started" id="submit" />
+    </form>
+
+    <div id="features">
+      <div class="features-item">
+        <div class="icon"><i class="fa-solid fa-fire-flame-curved"></i></div>
+        <div class="content">
+          <h1>Premium Materials</h1>
+          <p>
+            Our trombones use the shiniest brass which is sourced locally. This
+            will increase the longevity of your purchase.
+          </p>
+        </div>
+      </div>
+      <div class="features-item">
+        <div class="icon"><i class="fa-solid fa-truck"></i></div>
+        <div class="content">
+          <h1>Fast Shipping</h1>
+          <p>
+            We make sure you recieve your trombone as soon as we have finished
+            making it. We also provide free returns if you are not satisfied.
+          </p>
+        </div>
+      </div>
+      <div class="features-item">
+        <div class="icon"><i class="fa-solid fa-battery-full"></i></div>
+        <div class="content">
+          <h1>Quality Assurance</h1>
+          <p>
+            For every purchase you make, we will ensure there are no damages or
+            faults and we will check and test the pitch of your instrument.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <div class="video-container" id="how-it-works">
+      <iframe
+        id="video"
+        width="560"
+        height="315"
+        src="https://www.youtube.com/embed/y8Yv4pnO7qc"
+        title="YouTube video player"
+        frameborder="0"
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+        allowfullscreen
+      ></iframe>
+    </div>
+    <div id="pricing">
+      <div class="pricing-item">
+        <div class="title">TENOR TROMBONE</div>
+        <div class="pricing-item-details">
+          <div class="price">$600</div>
+          <p>
+            Lorem ipsum.<br />Lorem ipsum.<br />Lorem ipsum dolor.<br />Lorem
+            ipsum.
+          </p>
+          <button>select</button>
+        </div>
+      </div>
+      <div class="pricing-item">
+        <div class="title">TENOR TROMBONE</div>
+        <div class="pricing-item-details">
+          <div class="price">$600</div>
+          <p>
+            Lorem ipsum.<br />Lorem ipsum.<br />Lorem ipsum dolor.<br />Lorem
+            ipsum.
+          </p>
+          <button>select</button>
```

Comment:
I think you should set the button attribute "type". The value of the "type" attribute instructs the browser to provide a regular button that has no associated action.